EventListener/Condition/Condition.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Cydrickn\OpenApiValidatorBundle\EventListener\Condition;
+
+
+
+use Symfony\Component\HttpFoundation\Request;
+
+/**
```

Comment:
Hi @akaffenberger ,

Maybe you can remove this one, since this repo is public
Every one can update this file so, everyone can be an author,
Its fine that this will be removed since the commit is still yours.

Same with the other files

Reply:
Yep will do, I forgot that Intellij auto includes these when I create the class